src/utils/update/mapper.utils.ts

```diff
@@ -16,6 +16,33 @@ export const mapDateOfBirth: DateOfBirthTypes = (dateOfBirth) => {
   } as InputDate;
 };
 
+export const mapInputDate = (date: string | undefined): InputDate => {
```

Comment:
Wondering why we need this new function as we already resume registration journeys with dates, so wouldn't the format be the same (and this kind of logic already exist somewhere)?

Reply:
It was needed as we needed to convert date_of_creation from company profile where it was defined as string whereas for Registration everything was retrieved from overseas-entities service and they were all defined as InputDate in sdk-node

Reply:
👍 thanks. Would be nice if API SDK consistently returned the same format for all dates but as it doesn't, guess we have no choice but to do this.